app/src/components/NoAccess/index.tsx

```diff
@@ -0,0 +1,78 @@
+import { Box, Heading, Text, Grid, GridItem } from "@chakra-ui/react"; // Import Grid and GridItem
+import { useTranslation } from "@/i18n/client";
+import { Trans } from "react-i18next";
+import React from "react";
+import Image from "next/image";
+import { NavigationBar } from "@/components/navigation-bar";
+
+const NoAccessPage = ({ lng, email }: { lng: string; email: string }) => {
```

Comment:
I am a bit confused about the purpose of this page - if the user doesn't belong to any inventories, shouldn't we redirect them to onboarding so they can start using the platform?

Or does this flow work differently with GTM?